ui-components/src/lists/PreferencesList.tsx

```diff
@@ -18,7 +18,10 @@ const getOrdinal = (n: number) => {
 }
 
 const PreferencesList = (props: PreferencesListProps) => {
-  const preferences = props.preferences.map((preference: Preference, index: number) => {
+  const filteredPreferences = props.preferences.filter(
+    (pref) => !pref.formMetadata?.hideFromListing
+  )
+  const preferences = filteredPreferences.map((preference: Preference, index: number) => {
```

Comment:
I'm not opposed to not chaining, since it can be harder to read, so just a suggestion:
```
const preferences = props.preferences
    .filter((preference: Preference) => !preference.formMetadata?.hideFromListing)
    .map((preference: Preference, index: number) => {}
```